api/fastapi_service/services.py

```
@@ -429,30 +496,37 @@ async def graph_from_ids(city_id : int, regions_ids : List[int], regions : GeoDa
 
 
 def record_obj_to_pprop(record):
-    return [record.id_point ,record.property ,record.value]
+    return [record.id_point, record.property, record.value]
 
 
 async def graph_from_poly(city_id, polygon):
-    bbox = polygon.bounds   # min_lon, min_lat, max_lon, max_lat
+    bbox = polygon.bounds  # min_lon, min_lat, max_lon, max_lat
 
     q = CityAsync.select().where(CityAsync.c.id == city_id)
```

Ruff (0.14.1)

`CityAsync` may be undefined, or defined from star imports

(F405)